src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day_suncalendar--month-length_31">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day_sun calendar--month-length_31">
```

That's why your calendar should be started from "sun", but it's not working